3scale/porta
# THREESCALE-8626: Add stripe description and cardholder name

**What this PR does / why we need it**:

This PR fixes two things:
- Adds a description of the Payment Intent - this is not visible in the 3scale UI / API, but will be seen in Stripe dashboard.
- Adds a field for Cardholder name in the developer portal. Again, this will be only visible in Stripe dashboard.

Both are required for Stripe account operating in India to accept international payments according to https://stripe.com/docs/india-accept-international-payments#valid-charges

**Which issue(s) this PR fixes** 

Fixes https://issues.redhat.com/browse/THREESCALE-8626

**Verification steps** 
With Stripe configured in the admin portal:

For the cardholder name part:
- Go to the Credit Card form in the developer portal and confirm there is Cardholder name in the form
- Fill in the data (correct Stripe test numbers) and verify in Stripe dashboard that the card is added with the entered name

For payment description part:
- Create an invoice on an account
- Issue and charge the invoice
- Verify that the payment has succeeded
- Verify in Stripe dashboard that the payment intent has a description

**Special notes for your reviewer**:

- Regarding the remaining Code Climate issues (duplication), I don't think it's work fixing it now. The code for Invoice Payment page needs refactoring to move it to React, and this duplication needs to be fixed during this refactoring (style alignment has been done as part of the PR)
